crates/rome_js_syntax/src/static_value.rs

```diff
@@ -98,8 +10,10 @@ pub enum StaticValue {
     Undefined(JsSyntaxToken),
     Number(JsSyntaxToken),
     BigInt(JsSyntaxToken),
-    String(QuotedString),
-    TemplateChunk(Option<JsSyntaxToken>),
+    /// The string can be empty.
```

Comment:
```suggestion
    // The string can be empty.
```

nit: It doesn't seem a doc comment relevant to the consumers